src/index.html

```diff
@@ -20,8 +20,433 @@
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+
+        <a
+          class="card__button"
+          data-qa="card-hover"
+          href="#"
+        >
+          BUY
+        </a>
+      </section>
+
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Apple computer"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+
+        <a
+          class="card__button"
+          data-qa="hover"
+          href="#"
+        >
+          BUY
+        </a>
+      </section>
+
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Apple computer"
+        />
+
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__subtitle">Product code: 195434</p>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-label">Price:</div>
+          <div class="card__price-value">$2,199</div>
+        </div>
+
+        <a
+          class="card__button"
+          data-qa="hover"
+          href="#"
+        >
+          BUY
+        </a>
+      </section>
+
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Apple computer"
```

Comment:
If you have name of the visible image than you can use its name. Always try to be as much descriptive as you can in alt attribute